tests/core/pyspec/eth2spec/test/eip4844/unittests/test_kzg.py

```diff
@@ -0,0 +1,27 @@
+
+import random
+from eth2spec.test.helpers.constants import (
+    EIP4844,
+    MINIMAL,
+)
+from eth2spec.test.context import (
+    with_phases,
+    spec_state_test,
+    with_presets,
+)
+
+
+def _create_blob(spec):
+    rng = random.Random(5566)
+    return spec.Blob([
+        rng.randint(0, spec.BLS_MODULUS)
```

Comment:
The test looks good, but counter-intuitively this should be `rng.randint(0, spec.BLS_MODULUS-1)` since `randint` [works on an inclusive range on both sides](https://stackoverflow.com/questions/2568783/python-why-does-random-randinta-b-return-a-range-inclusive-of-b)...

There is no way this could ever be a problem, but just mentioning it for correctness.